SensusUI/Inputs/NumberSliderInput.cs

```diff
@@ -172,48 +240,69 @@ public override View GetView(int index)
                     {
                         _incrementalValue = newIncrementalValue;
                         _incrementalValueHasChanged = true;
-                        _sliderLabel.Text = GetLabelText(index) + "  " + _incrementalValue;
+                        _sliderLabel.Text = DisplaySliderValue ? GetLabelText(index) + "  " + _incrementalValue : GetLabelText(index) + "  " + _tipText;
                         Complete = Value != null;
                     }
                 };
 
-                base.SetView(new StackLayout
-                    {
-                        Orientation = StackOrientation.Vertical,
-                        VerticalOptions = LayoutOptions.Start,
-                        Children =
-                        { 
-                            _sliderLabel,
-                            new StackLayout
+                StackLayout stack = new StackLayout();
+                stack.Orientation = StackOrientation.Vertical;
+                stack.VerticalOptions = LayoutOptions.Start;
+                stack.Children.Add(_sliderLabel);
```

Comment:
I tried using the ternary conditional to set the StackLayout's children within an initializer block, but I couldn't get the syntax to work. Stackoverflow says this is possible for simple statements, but I haven't found any examples for {} statements like the one necessary to set children. Is this correct? If so should I set up 4 initializer blocks within their own if statements to cover the 4 possible permutations of the conditions (_displayMinMax) and (_leftLabel != null || _rightLabel != null)?